packages/cli/src/commands/deploy.ts

```diff
@@ -40,73 +40,239 @@ import {
+import { help } from './help';
+import type { Command } from './help';
+
+const deployCommand: Command = {
+  name: 'deploy',
+  description:
+    'Deploy your project to Vercel. The `deploy` command is the default command for the Vercel CLI, and can be omitted (`vc deploy my-app` equals `vc my-app`).',
+  arguments: [
+    {
+      name: 'project-path',
+      required: false,
+    },
+  ],
+  options: [
+    {
+      name: 'force',
+      shorthand: 'f',
+      type: 'boolean',
+      deprecated: false,
+      description: 'Force a new deployment even if nothing has changed',
+      multi: false,
+    },
+    {
+      name: 'with-cache',
+      shorthand: null,
+      type: 'boolean',
+      deprecated: false,
+      description: 'Retain build cache when using "--force"',
+      multi: false,
+    },
+    {
+      name: 'public',
+      shorthand: 'p',
+      type: 'boolean',
+      deprecated: false,
+      description: 'Deployment is public (`/_src`) is exposed)',
+      multi: false,
+    },
+    {
+      name: 'env',
+      shorthand: 'e',
+      type: 'string',
+      argument: 'key=value',
+      deprecated: false,
+      multi: true,
+      description:
+        'Specify environment variables during run-time (e.g. `-e KEY1=value1 -e KEY2=value2`)',
+    },
+    {
+      name: 'build-env',
+      shorthand: 'b',
+      type: 'string',
+      argument: 'key=value',
+      deprecated: false,
+      multi: true,
+      description:
+        'Specify environment variables during build-time (e.g. `-b KEY1=value1 -b KEY2=value2`)',
+    },
+    {
+      name: 'meta',
+      shorthand: 'm',
+      type: 'string',
+      argument: 'key=value',
+      deprecated: false,
+      multi: true,
+      description:
+        'Specify metadata for the deployment (e.g. `-m KEY1=value1 -m KEY2=value2`)',
+    },
+    {
+      name: 'regions',
+      shorthand: null,
+      type: 'string',
+      deprecated: false,
+      description: 'Set default regions to enable the deployment on',
+      multi: false,
+    },
+    {
+      name: 'prebuilt',
+      shorthand: null,
+      type: 'boolean',
+      deprecated: false,
+      description: 'TODO: Description for --prebuilt',
+      multi: false,
+    },
+    {
+      name: 'prod',
+      shorthand: null,
+      type: 'boolean',
+      deprecated: false,
+      description: 'Create a production deployment',
+      multi: false,
+    },
+    {
+      name: 'archive',
+      shorthand: null,
+      type: 'string',
+      deprecated: false,
+      description: 'TODO: Description for --archive',
+      multi: false,
+    },
+    {
+      name: 'no-wait',
+      shorthand: null,
+      type: 'boolean',
+      deprecated: false,
+      description: "Don't wait for the deployment to finish",
+      multi: false,
+    },
+    {
+      name: 'skip-domain',
+      shorthand: null,
+      type: 'boolean',
+      deprecated: false,
+      description: 'TODO: Description for --skip-domain',
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏leave undocumented

If you accept my other suggestion to filter out undocumented (empty description) options, can we clear out the description here? This is intentionally undocumented until we can provide guidance on the full updated workflow.

```suggestion
      description: undefined,
```